tests/api2/test_480_system_advanced.py

```diff
@@ -93,12 +93,26 @@ def test_10_system_advanced_check_motd_using_ssh(request):
     assert results['result'] is True, results
 
 
-def test_11_Setting_sysloglevel():
+def test_11_system_advanced_login_banner():
+    results = PUT('/system/advanced/', {
+        'login_banner': 'TrueNAS login banner.'
+    })
+    assert results.status_code == 200, results.text
+    results = GET('/system/advanced')
+    assert results.status_code == 200, results.text
+    data = results.json()
+    assert data['login_banner'] is True
+
+    results = SSH_TEST('cat /etc/ssh/sshd_config | grep Banner', user, password)
```

Comment:
You can just `grep Banner /etc/ssh/sshd_config`

Reply:
We can probably opportunistically clean up this test file while you're here.

1. remove test numbers
2. convert REST to WS calls

Reply:
I was following the paradigm from earlier in the tests, I'll just directly grep lol
I'll also update that earlier one too

Reply:
In regards to converting REST to WS calls, should we still be checking that a dict is being returned or is that guaranteed with our WS calls?